Tests/test_file_tga.py

```diff
@@ -60,8 +57,16 @@ def test_save_rle(self):
         test_im = Image.open(test_file)
         self.assertEqual(test_im.size, (199, 199))
 
-        # Unsupported mode save
-        self.assertRaises(IOError, lambda: im.convert("LA").save(test_file))
+    def test_save_l_transparency(self):
+        in_file = "Tests/images/l_trns.tga"
+        im = Image.open(in_file)
+
+        test_file = self.tempfile("temp.tga")
+        im.save(test_file)
+
+        # There are 559 transparent pixels.
+        im = im.convert('RGBA')
+        self.assertEqual(im.getchannel('A').getcolors()[0][0], 559)
```

Comment:
* This test opens l_trns.tga and saves it to a temp file.
* It then converts the image to RGBA and asserts the number of pixels.

It's good it's making sure the save runs without exception, but it doesn't verify the save.

How about opening the saved file and asserting something there? Have a look at the other tests here.